go.mod

```diff
@@ -6,4 +6,4 @@ require (
     github.com/pingcap/badger v1.5.1-0.20200908111422-2e78ee155d19
 )
 
-replace github.com/pingcap/badger => github.com/spongedu/badger v1.5.1-0.20210114105910-471ed28fdea2
+replace github.com/pingcap/badger => ../badger
```

Comment:
I think it is nice to use a local repo, so you can do your changes on `badger` then run the code here to see the difference.